Makefile

```diff
@@ -94,17 +96,17 @@ clean:
 .PHONY: images
 images:
 	@echo "VERSION: ${RELEASE_VERSION}"
-	${IMG_BUILDER} build --platform $(PLATFORMS) --build-arg VERSION=${CONTROLLER_VERSION} -t ${IMAGE_NAME}:${RELEASE_VERSION} .
+	${IMG_BUILDER} build $(BUILD_ARGS) --platform $(PLATFORMS) --build-arg VERSION=${CONTROLLER_VERSION} --build-arg RELEASE_VERSION=${RELEASE_VERSION} -t ${IMAGE_NAME}:${RELEASE_VERSION} .
 
 .PHONY: test_images
 test_images:
-	${IMG_BUILDER} build --platform $(PLATFORMS) --build-arg port=${BASE_IMAGE_SSH_PORT} -t mpioperator/base build/base
-	${IMG_BUILDER} build --platform $(PLATFORMS) -t mpioperator/openmpi build/base -f build/base/openmpi.Dockerfile
-	${IMG_BUILDER} build --platform $(PLATFORMS) -t mpioperator/openmpi-builder build/base -f build/base/openmpi-builder.Dockerfile
-	${IMG_BUILDER} build --platform $(PLATFORMS) -t mpioperator/mpi-pi:openmpi examples/v2beta1/pi
-	${IMG_BUILDER} build --platform $(PLATFORMS) -t mpioperator/intel build/base -f build/base/intel.Dockerfile
-	${IMG_BUILDER} build --platform $(PLATFORMS) -t mpioperator/intel-builder build/base -f build/base/intel-builder.Dockerfile
-	${IMG_BUILDER} build --platform $(PLATFORMS) -t mpioperator/mpi-pi:intel examples/v2beta1/pi -f examples/v2beta1/pi/intel.Dockerfile
+	${IMG_BUILDER} build $(BUILD_ARGS) --platform $(PLATFORMS) --build-arg port=${BASE_IMAGE_SSH_PORT} -t mpioperator/base:${RELEASE_VERSION} build/base
```

Comment:
Do we actually use `BUILD_ARGS`? It doesn't seem to use that env vars in the whole Makefile.

Reply:
They can be used to directly push like this: `docker buildx build --push`.

Reply:
Ah, I see. Thanks for clarifying!